lib/errors.js

```diff
@@ -129,6 +129,35 @@ function createInvalidExceptionError(message, value) {
   return err;
 }
 
+/**
+ * Creates an error object to be thrown when a mocha object's `run` method is executed while it is already disposed.
+ * @param {string} message The error message to be displayed.
+ * @param {boolean} cleanReferencesAfterRun the value of `autoDispose`
```

Comment:
this references `autoDispose` but I don't see this name elsewhere in the file.  should it be `cleanReferencesAfterRun`?